flujo/state/backends/base.py

```
@@ -80,6 +80,44 @@ async def get_failed_workflows(self, hours_back: int = 24) -> List[Dict[str, Any
+                wf
+                for wf in background_tasks
+                if wf.get("metadata", {}).get("parent_run_id") == parent_run_id
+                or wf.get("parent_run_id") == parent_run_id
+            ]
+
+        return background_tasks
+
+    async def get_failed_background_tasks(
+        self,
+        parent_run_id: Optional[str] = None,
+        hours_back: int = 24,
```

Ruff (0.14.6)

Unused method argument: `hours_back`

(ARG002)